README.md

```diff
@@ -58,331 +63,232 @@ az login
-# you only need to remember the name of the env file (or set the $mikv_Name variable again)
-source ~/{yoursameuniquename}.env
+# you only need to remember the name of the env file
+source ~/YourUniqueName.env
 
 ```
 
-Create Azure Key Vault
+### Create Azure Key Vault
 
-* All secrets are stored in Azure Key Vault for security
-  * This sample uses Managed Identity to access Key Vault
+- All secrets are stored in Azure Key Vault for security
+  - Use System Managed Identity to access Key Vault
 
 ```bash
 
-## Create the Key Vault and add secrets
-az keyvault create -g $mikv_App_RG -n $mikv_Name
+## create the Key Vault
+az keyvault create -g $MIKV_RG -n $MIKV_NAME
 
 # add a secret
-az keyvault secret set -o table --vault-name $mikv_Name --name "MySecret" --value "Hello from Key Vault and Managed Identity!"
-
-```
-
-(Optional) In order to run the application locally, each developer will need access to the Key Vault. Since you created the Key Vault during setup, you will automatically have permission, so this step is only required for additional developers.
-
-Use the following command to grant permissions to each developer that will need access.
-
-```bash
-# get the object id for each developer (optional)
-export dev_Object_Id=$(az ad user show --id {developer email address} --query objectId -o tsv)
-
-# grant Key Vault access to each developer (optional)
-az keyvault set-policy -n $mikv_Name --secret-permissions get list --key-permissions get list --object-id $dev_Object_Id
-
-```
-
-Run the unit tests
-
-```bash
-
-cd src/unit-tests
-
-dotnet test --logger:trx
+az keyvault secret set \
+  --vault-name $MIKV_NAME \
+  --name "MySecret" \
+  --value "Hello from Key Vault and Managed Identity"
 
 ```
 
-Run the application locally
-
-```bash
-
-cd ../app
-
-# run in the background
-dotnet run $mikv_Name &
-
-# the application takes about 10 seconds to start
-# wait for the web server started message
-curl http://localhost:4120/healthz
-
-```
-
-(Alternative) Run the application as a local container
+### Create Azure Container Registry
 
+> Create the ACR with admin access `disabled` for security
 
 ```bash
 
-# make sure you are in the src folder
-cd ..
+# create the ACR
+az acr create --sku Standard --admin-enabled false -g $MIKV_RG -n $MIKV_NAME
 
-# build the dev image
-# you may see red warnings in the build output, they are safe to ignore
-# examples: "debconf: ..." or "dpkg-preconfigure: ..."
-docker build -t mikv-dev -f Dockerfile-Dev .
+# get the ACR_ID
+export MIKV_ACR_ID=$(az acr show -g $MIKV_RG -n $MIKV_NAME --query id --output tsv)
 
-# run the container
-# mount your ~/.azure directory to container root/.azure directory
-docker run -d -p 4120:4120 --name mikv-dev -v ~/.azure:/root/.azure mikv-dev "dotnet" "run" "${mikv_Name}"
+# login to ACR
+# if you get an error that the login server isn't available,
+#   it's a DNS issue that will resolve in a minute or two, just retry
+az acr login -n $MIKV_NAME --expose-token
 

```

Comment:
Why was the --expose-token added?

Reply:
that allows the az acr commands to work without a local dockerd running which allows everything to work in Cloud Shell